src/bench/mempool_stress.cpp

```diff
@@ -79,6 +76,17 @@ static void ComplexMemPool(benchmark::Bench& bench)
         ordered_coins.emplace_back(MakeTransactionRef(tx));
         available_coins.emplace_back(ordered_coins.back(), tx_counter++);
     }
+    return ordered_coins;
+}
+
+static void ComplexMemPool(benchmark::Bench& bench)
+{
+    FastRandomContext det_rand{true};
+    int childTxs = 800;
+    if (bench.complexityN() > 1) {
+        childTxs = static_cast<int>(bench.complexityN());
+    }
```

Comment:
This could be simplified to: 
`const int childTxs = bench.complexityN() > 1 ? static_cast<int>(bench.complexityN()) : 800;`
as it is on line 105